src/main/java/org/mqttbee/mqtt/mqtt3/Mqtt3ClientDataView.java

```diff
@@ -55,11 +57,27 @@ public int getServerPort() {
         return wrapped.getServerPort();
     }
 
+    @Override
+    public String getServerPath() {
+        return wrapped.getServerPath();
+    }
+
     @Override
     public boolean usesSSL() {
         return wrapped.usesSSL();
     }
 
+    @Nullable
+    @Override
+    public KeyStore getKeyStore() { return wrapped.getKeyStore(); }
```

Comment:
This is also simpler if we have a MqttClientSslConfig and -WebsocketConfig

Reply:
done